pkg/nsx/services/vpc/builder.go

```diff
@@ -45,6 +46,26 @@ func TransferIpblockIDstoPaths(ids []string) []string {
 	return paths
 }
 
+func buildPrivateIpBlock(vpc *v1alpha1.VPC, cidr string, ip string, project string, cluster string) model.IpAddressBlock {
+	suffix := vpc.GetNamespace() + "-" + string(vpc.GetUID())
+	blockId := util.GenerateID(cidr, "", suffix, "")
+	suffix = ip + "-" + vpc.GetNamespace() + "-" + string(vpc.GetUID())
```

Comment:
ip -> cidr

Reply:
not suggest to use cidr here because cidr contains a "/"